scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1281,24 +1310,56 @@ class Router(formatOps: FormatOps) {
         }
       case FormatToken(_: T.Comma, right, _) if leftOwner.isNot[Template] =>
         // TODO(olafur) DRY, see OneArgOneLine.
```

Comment:
Does the comment still make sense? Without an issue connected to it it's hard to figure out what this means exactly.

Reply:
removed